### PR TITLE
fix: installed module can not get arguments correctly

### PR DESCRIPTION
**Pull request by axetroy**
The generated command is not wrapped in double quotes

This will cause the parameters passed to the script to be incorrect

change:

```diff
- deno run --allow-net --allow-read https://deno.land/std/http/file_server.ts hello world "$@"
+ "deno" "run" "--allow-net" "--allow-read" "https://deno.land/std/http/file_server.ts" "hello world" "$@"
```

### CODE REVIEW

**Comment by axetroy** [comment]
/cc @ry 
sorry. my mistake.

File: installer/mod.ts [hunk 1]
```diff
@@ -153,6 +153,7 @@ async function generateExecutable(
   filePath: string,
   commands: string[]
 ): Promise<void> {
+  commands = commands.map((v): string => '"' + v + '"');
```
Comment (ry) [inline installer/mod.ts]: Do these strings need to be escaped too? What if they contain a quote mark?

Comment (axetroy) [reply installer/mod.ts]: yes. I did it.

File: installer/mod.ts [hunk 2]
```diff
@@ -153,7 +153,7 @@ async function generateExecutable(
   filePath: string,
   commands: string[]
 ): Promise<void> {
-  commands = commands.map((v): string => '"' + v + '"');
+  commands = commands.map((v): string => `"${v.replace(/\"/g, '\\"')}"`);
```
Comment (ry) [inline installer/mod.ts]: Hrm... I don't think this is sufficient to escape the strings properly. (I can't immediately think of a counter example...) What about using JSON.stringify() on the strings?

Comment (axetroy) [reply installer/mod.ts]: yes, it works.

Comment (ry) [reply installer/mod.ts]: You can leave out the anonymous function I think.

Comment (axetroy) [reply installer/mod.ts]: @ry No, it can't. typescript will throw an error.

**Pull request merged**